resources/lib/mvupdate.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import argparse
 import datetime
-import defusedxml.ElementTree as ET
+from xml.etree import ElementTree as ET
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using ElementTree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace ElementTree with the equivalent defusedxml package, or make sure defusedxml.defuse_stdlib() is called.](https://app.codacy.com/manual/MediathekView/plugin.video.mediathekview/pullRequest?prid=5447573)